registry/dbConfig/postgres/schemas/auth-basic.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE users (
+    "id" SERIAL,
+    "user_id" UUID NOT NULL DEFAULT gen_random_uuid(),
```

Comment:
Could see this potentially get confusing as it is ambiguous if I ask the question "get me the user with id = XXXX". Maybe we omit `user_id`?

Reply:
I am not sure tbh, you know the reason to have a serial ID and also the globally unique UUID column, but I am not sure if the developer would know it.. Do we need to explain this in the docs or just remove `user_id`?